guide/src/credentials/id_tokens.md

```diff
@@ -0,0 +1,131 @@
+<!--
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# ID Tokens
+
+This guide shows you how to generate, use and verify [OIDC ID tokens] using the
+`google-cloud-auth` crate.
+
+ID tokens are a standardized way to verify the identity of a principal in a
+secure and portable manner. Unlike access tokens, which are used to authorize
+access to Google Cloud APIs, ID tokens are used for service-to-service
+authentication. The requesting service can generate an ID token and include it
+in the `Authorization` header of a request to the receiving service. The
+receiving service can then verify the token to authenticate the caller.
+
+ID tokens are particularly useful in scenarios where you need to authenticate to
+a service that is not a Google Cloud API. For example, if you have a service
+running on Cloud Run or behind an Identity-Aware Proxy (IAP), you can use ID
+tokens to securely authenticate requests between your services.
```

Comment:
```suggestion
a service that is not a Google Cloud API. For example, you can use ID
tokens to securely authenticate requests if you the target service is
running on Cloud Run or behind an Identity-Aware Proxy.
```

The acronym is not used again, removed. Also link Cloud Run and Identity-Aware proxy to their main sites.

Reply:
done